package_repo_resource.md

```diff
@@ -0,0 +1,71 @@
+---
+RFC: unassigned
+Author: Thom May <tmay@chef.io>
+Status: Draft
+Type: Standards Track
+---
+
+# Add a resource to manage package repositories
+
+Provide a core resource to allow a user to add a package repository to
+their system.
+
+## Motivation
+
+    As a cookbook author,
+    I want to write cookbooks that rely only on core resources,
+    so that I can minimise my dependencies.
+
+## Specification
+
+Create a new `package_repo` resource, similar to 
+```
+package_repo "foo" do
+  description "A descriptive string"
+  base_url "https://some/url"
+  gpgkey "https://some/other/url"
```

Comment:
probably everything on Linux will use GPG, but are there any other cases which use non-gpg signatures (anyone know how Windows nu-get works)? What I'm getting at is, should this just be `key` ?